src/_includes/templates/forms/submit-a-request-form.html

```diff
@@ -1,62 +1,21 @@
+    {% import 'macros/form.html' as form %}
+    {% from 'common-form-fields.html' import state as state %}
+    {% from 'common-form-fields.html' import date_picker as date_picker %}
```

Comment:
Consolidate if https://github.com/cfpb/cfgov-refresh/issues/938 is possible. If it isn't possible close 938.